src/Symfony/Component/VarDumper/composer.json

```diff
@@ -29,7 +29,8 @@
     },
     "suggest": {
         "ext-iconv": "To convert non-UTF-8 strings to UTF-8 (or symfony/polyfill-iconv in case ext-iconv cannot be used).",
-        "ext-intl": "To show region name in time zone dump"
+        "ext-intl": "To show region name in time zone dump",
+        "ext-gmp": "To dump GMP resources to string or integer"
```

Comment:
this does not make any sense: the VarDumper component does not provide any feature *needing* gmp. It would just improve the dumping when *your own code* uses gmp. So there is no point suggesting it. It would just be spam in the suggestions

Reply:
i'll remove it